packages/zowe-explorer-api/src/fs/types/datasets.ts

```diff
@@ -47,8 +47,18 @@ export class DsEntryMetadata implements EntryMetadata {
         this.path = metadata.path;
     }
 
+    private extensionRemovedFromPath(): string {
+        for (const ext of [".c", ".jcl", ".cbl", ".cpy", ".inc", ".pli", ".sh", ".rexx", ".xml", ".asm", ".log"]) {
+            if (this.path.endsWith(ext)) {
+                return this.path.replace(ext, "");
+            }
+        }
```

Comment:
should we loop through the list, or just remove the last extension if found in the path?

example:
`test.log.rexx.jcl` will result in `test` 😢 

I know that's a bit unrealistic, but how about a dataset that contains log related information for a JCL job that executes REXX 😅 

`HLQ.LOG.REXX.JCL`

Reply:
I missed the return statement 😅